### DISCUSSION TITLE
How to send hid information to the mouse

### DISCUSSION THREAD

**Discussion by FBMNBAWA** [adafruit/Adafruit_TinyUSB_Arduino#463] [Q&A]
How can I use rp2040 to send messages to my mouse with my mouse plugged in?Can you give me an example

**Comment by pschatzmann** [comment]
I am totally confused about your question which does not make any sense to me.
This library provides support for impmenting a __[HID device](https://github.com/adafruit/Adafruit_TinyUSB_Arduino/blob/master/src/arduino/hid/Adafruit_USBD_HID.h)__: so you can implement a mouse which would send information to your PC.
